.github/scripts/label_promoted_commits.py

```diff
@@ -57,8 +70,25 @@ def main():
             }
             response = requests.get(search_url, headers=headers, params=params)
             prs = response.json().get("items", [])
-            for pr in prs:
-                refs_match = re.findall(r'Refs (?:#|https.*?)(\d+)', pr["body"])
+            if prs:
+                for pr in prs:
+                    refs_match = re.findall(r'Refs (?:#|https.*?)(\d+)', pr["body"])
+                    ref = re.search(r'branch-(\d+\.\d+)', args.ref)
```

Comment:
the variable names of `refs_match` and `refs` are confusing. could you please use something like `refs_issue` and `target_branch`?